enterprise/payara.tooling/nbproject/project.xml

```diff
@@ -71,7 +80,7 @@
                 </test-type>
             </test-dependencies>
             <friend-packages>
-                <friend>org.netbeans.modules.payara.cloud</friend>
+                <friend>org.netbeans.modules.payara.micro</friend>
```

Comment:
Removing from list of friends is an incompatible API change.

Reply:
Payara Modules are originally forked from GlassFish modules, `org.netbeans.modules.payara.cloud` listing is caused by replacing org.netbeans.modules.glassfish -> org.netbeans.modules.payara package, which actually does not exist, so removed it.